roles/postgres/templates/postgres.deployment.yaml

```diff
@@ -22,12 +22,12 @@ spec:
             - name: POSTGRESQL_DATABASE
               value: pulp
             - name: POSTGRESQL_USER
-              value: "{{ database_connection.username }}"
+              value: "{{ database_settings.username }}"
             - name: POSTGRESQL_PASSWORD
-              value: "{{ database_connection.password }}"
-{% if database_connection.admin_password is defined %}
+              value: "{{ database_settings.password }}"
+{% if database_settings.postgres_user_password is defined %}
             - name: POSTGRESQL_ADMIN_PASSWORD
-              value: "{{ database_connection.admin_password }}"
+              value: "{{ database_settings.postgres_user_password }}"
```

Comment:
Also, was it for interface consistency with ansible-pulp that we switched from `password` & `admin_password` to `password` and `postgres_user_password`?

Reply:
This was switched because I did not think that 'database_connection' was teh right description for settings that controlled which accounts/passwords are created in the postgresql container. 

What do you think?